src/stuf/migrations/0010_auto_20220404_1053.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.2.12 on 2022-04-04 08:53
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    SoapService = apps.get_model("stuf", "SoapService")
+    StufService = apps.get_model("stuf", "StufService")
+
+    updated_soap_services = []
+    for stuf_service in StufService.objects.all():
+        soap_service = stuf_service.soap_service
+        soap_service.endpoint_security = stuf_service.endpoint_security
+        soap_service.user = stuf_service.user
+        soap_service.password = stuf_service.password
+        soap_service.client_certificate.public_certificate = stuf_service.certificate
+        soap_service.client_certificate.private_key = stuf_service.certificate_key
+        soap_service.soap_version = stuf_service.soap_version
+        updated_soap_services.append(soap_service)
+
+    SoapService.objects.bulk_update(
+        updated_soap_services,
+        [
+            "endpoint_security",
+            "user",
+            "password",
+            "client_certificate",
+            "soap_version",
+        ],
+    )
```

Comment:
Fairly certain that this does not save the `soap_service.client_certificate` changes.